### PR TITLE
HostGame model added

### PR DESCRIPTION
**Pull request by sudhakarvelagala**

### CODE REVIEW

**Comment by sudhakarvelagala** [comment]
Capture PNG5666

File: users/models.py [hunk 1]
```diff
@@ -21,5 +21,25 @@ class profileModel(models.Model):
...
+        return "{}".format(self.about)
+
+
+
+class HostGame(models.Model):
+    title=models.CharField(max_length=20)
+    description=models.TextField()
+    image=models.ImageField()
+    game=models.ForeignKey(Game,on_delete=models.CASCADE)
+    players_team=models.CharField(max_length=20)
```
Comment (rajeshrapally) [inline users/models.py:33]: players_team should be a choice field

**Comment by sudhakarvelagala** [comment]
Capture PNG300

**Review by rajeshrapally — APPROVED** [review]

**Review by nagpursai — APPROVED** [review]
LGTM

**Pull request merged**